packages/edit-post/src/components/layout/style.scss

```diff
@@ -17,6 +17,7 @@
 	position: fixed;
 	right: 0;
 	bottom: 20px;
+	margin-bottom: 16px;
```

Comment:
Have kept this to be 16px to be consistent with the other paddings. Happy to take feedback from design team.

Reply:
The breadcrumb div's height is `24px`, might be better to use that as an offset rather than the existing padding values.

Also, any reason we're introducing `margin-bottom` rather than adding to the existing `bottom` property?

Reply:
Just `24px` will stick the Snackbar to the breadcrumb div. We will need some breathing space.

Though, good call on the bottom property. Updated it. e9ca9d1